<launch>
  <arg name="robot_name" default="r0"/>
  <arg name ="distance_threshold" default="0.2"/>
  <arg name ="obstacle_vote_threshold" default="5"/>


<group ns="$(arg robot_name)">
  <node pkg="nodelet" type="nodelet" name="CollisionAlarmNodeletManager"  args="manager" output="screen"/>


  <node pkg="nodelet" type="nodelet" name="CollisionAlarmNodelet" args="load tuw_collision_alarm/CollisionAlarmNodelet CollisionAlarmNodeletManager" output="screen">
    <param name="distance_threshold" value="$(arg distance_threshold)" />
    <param name="obstacle_vote_threshold" value= "$(arg obstacle_vote_threshold)" />

  </node>

  </group>
</launch>